compiler_opt/rl/imitation_learning/generate_bc_trajectories_lib.py

```diff
@@ -918,6 +927,10 @@ def gen_trajectories(
     worker_manager_class: A pool of workers hosted on the local machines, each
       in its own process.
   """
+  if (None in (_BASE_PATH.value, FLAGS.keep_temps) and
+      not all(el is None for el in (_BASE_PATH.value, FLAGS.keep_temps))):
+    raise ValueError(('Both flags keep_temps={FLAGS.keep_temps} and'
+                      'base_path={_BASE_PATH.value} should be set or be None'))
```

Comment:
Is there a reason that both of these have to be set? I think it would make sense to turning on keeping temps if either of the flags is set rather than requiring both.

(Sorry if I missed some conversation about this on the API design side).

Reply:
I think keep_temps can be set independently of base_path but if base path is set keep_temps should be set, so I will do this check instead.